# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/EarlyHintsService.cpp
# Commit: 8575cd92e1d8
# Full Hash: 8575cd92e1d8cd783e6cca3f6feadef11f0c4cc8
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 8: Add list of open channels from OngoingEarlyHints to RedirectToRealChannelArgs r=necko-reviewers,valentin,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161179
# ==============================================================================

diff -r ce8f0a8fa688 -r 8575cd92e1d8 netwerk/protocol/http/EarlyHintsService.cpp
--- a/netwerk/protocol/http/EarlyHintsService.cpp	Fri Dec 02 16:15:55 2022 +0000
+++ b/netwerk/protocol/http/EarlyHintsService.cpp	Fri Dec 02 16:15:56 2022 +0000
@@ -100,6 +100,11 @@
   }
 }
 
+void EarlyHintsService::RegisterLinksAndGetConnectArgs(
+    nsTArray<EarlyHintConnectArgs>& aOutLinks) {
+  mOngoingEarlyHints->RegisterLinksAndGetConnectArgs(aOutLinks);
+}
+
 void EarlyHintsService::CollectTelemetry(Maybe<uint32_t> aResponseStatus) {
   // EH_NUM_OF_HINTS_PER_PAGE is only collected for the 2xx responses,
   // regardless of the number of received mEarlyHintsCount.